package.json

```diff
@@ -449,12 +449,12 @@
         },
         {
           "command": "maven.project.excludeDependency",
-          "when": "view == mavenProjects && viewItem == Dependency",
+          "when": "viewItem == Dependency || viewItem == conflictDependency && view == mavenProjects",
```

Comment:
For `||` and `&&`, which operator has higher priority?

Reply:
oops, wrong understanding but get right results.
I have tried `"view == mavenProjects && viewItem =~ /(conflict){0,1}Dependency/"` but it doesn't work. What is the correct way to represent `"Dependency || conflictDependency"` ?

Reply:
now use `view == mavenProjects && viewItem =~ /Dependency | conflictDependency/` and it works.